awscli/examples/rum/create-app-monitor.rst

```diff
@@ -0,0 +1,15 @@
+**To create a Amazon CloudWatch RUM app monitor**
+
+The following ``create-app-monitor`` creates a Amazon CloudWatch RUM app monitor. ::
+
+	aws rum create-app-monitor \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general